ziti-fabric/subcmd/list_links.go

```diff
@@ -58,22 +58,24 @@ func listLinks(cmd *cobra.Command, args []string) {
 				err := proto.Unmarshal(responseMsg.Body, response)
 				if err == nil {
 					out := fmt.Sprintf("\nLinks: (%d)\n\n", len(response.Links))
-					out += fmt.Sprintf("%-6s | %-24s -> %-24s | %-12s | %-4s | %-12s\n", "Id", "Src", "Dst", "State", "Cost", "Latency")
+					out += fmt.Sprintf("%-6s | %-24s -> %-24s | %-12s | %-4s | %-13s | %-12s | %-6s\n", "Id", "Src", "Dst", "State", "Cost", "Latency", "Full Cost", "Status")
 					sort.Slice(response.Links, func(i, j int) bool {
 						if response.Links[i].Src == response.Links[j].Src {
 							return response.Links[i].Dst < response.Links[j].Dst
 						}
 						return response.Links[i].Src < response.Links[j].Src
 					})
 					for _, l := range response.Links {
-						down := ""
+						status := "up"
 						if l.Down {
-							down = "->down<-"
+							status = "down"
 						}
 						// Convert nanoseconds to fractional seconds
-						srcLatency := float64(l.SrcLatency) / 1000000000.0
-						dstLatency := float64(l.DstLatency) / 1000000000.0
-						out += fmt.Sprintf("%-6s | %-24s -> %-24s | %-12s | %-4d | %-0.4f %-0.4f %s\n", l.Id, l.Src, l.Dst, l.State, l.Cost, srcLatency, dstLatency, down)
+						srcLatency := float64(l.SrcLatency) / 1_000_000_000.0
```

Comment:
i still like constants over magic numbers but maybe that's just me? :)

Reply:
I'm not sure `float64(l.SrcLatency) / OneBillion` would provide any additional clarity :)

`float64(l.SrcLatency) / NanosecondsToSecondsScaleFactor` might, but I think the comment directly above does the same